apps/dashboard/src/main/resources/inbuilt_test_yaml_files/LikeDislikeRaceCondition.yml

```diff
@@ -0,0 +1,67 @@
+id: LIKE_DISLIKE_RACE_CONDITION
+info:
+  name: Race condition in like/dislike API
+  description: Attacker can like/dislike a video multiple times by exploiting race condition vulnerability
+  details: > 
+    "In a normal scenario, the user will be able to like or dislike video or comment, only after all other data was received.
+    However, re-submitting the same request right after the original request, might end up in like or dislike for comment
+    or video multiple time."
+  impact: "A typical Time of Check Time of Use vulnerability, allowing an attacker to steal from the application by submitting out-of-order requests"
+  category:
+    name: BLE
+    shortName: Business Logic Errors
+    displayName: Business Logic Errors (BLE)
+  subCategory: LIKE_DISLIKE_RACE_CONDITION
+  severity: MEDIUM
+  tags:
+    - Business logic
+    - OWASP Top 10
+  references:
+    - "https://github.com/OWASP/DVSA/blob/c04e0a536ac54db8f732144be92b5f208169f8c4/CONTENT/AWS/LESSONS/LESSON_08.md#81-race-condition"
+
+api_selection_filters:
+  response_code:
+    gte: 200
+    lt: 300
+  or:
+    url:
+      regex: .*(like|vote).*
+      extract: urlVar
+        
+    query_param:
+      for_one:
+        key:
+          contains_either:
+            - like
+            - dislike
+            - vote
+        value:
+          extract: queryVar
+    request_payload:
+      for_one:
+        key:
+          contains_either:
+            - like
+            - dislike
+            - vote
+        value:
+          extract: bodyVar
+
+execute:
+  type: single
+  requests:
+    - req:
+      - modify_url: ${urlVar}
+      - modify_query_param: 
+          queryVar: ${queryVar}
+      - modify_body_param: 
+          bodyVar: ${bodyVar}
```

Comment:
didn't understand what's going on in this

Reply:
i added a check for like count